docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -0,0 +1,108 @@
+---
+id: plugin-types-usage
+title: Plugin types and usage
+description: Learn about the different types of Grafana plugins, their usage and configuration.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - usage
+  - provisioning
+  - provision
+  - configuration
+  - configure
+sidebar_position: 1
+---
+
+# Grafana plugin types and usage
+
+:::note
+
+This documentation discusses plugin configuration, provisioning, and usage for plugin developers. General information about plugin administration is available at [Plugin management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
+
+::: 
+## Overview
+
+Grafana plugin development allows for many options depending on the type of user experience you want to create. Whatever your need, there's a SDK-supported plugin type for your intended use:
+- **Data-source plugin** - a connection to a new database or other source of data,
```

Comment:
Below you use _Data source plugins_ and here _Data-source plugins_. What do we use?

```suggestion
- **Data source plugin** - a connection to a new database or other source of data,
```

Reply:
For what it's worth, this topic has been brought up again recently in the docs team and we intend to socialize the change or revert it.

Reply:
Okay got it :+1: